build.gradle

```diff
@@ -50,66 +50,53 @@ springBoot {
     }
 }
 
-// Apply application plugin for easy application start
-apply plugin: 'application'
-
-mainClassName = "uk.gov.justice.digital.nomis.CustodyApiApplication"
-
-// Apply the groovy plugin to also add support for Groovy (needed for Spock)
-apply plugin: 'groovy'
-
-// In this section you declare where to find the dependencies of your project
-repositories {
-    // Use 'jcenter' for resolving your dependencies.
-    // You can declare any Maven/Ivy/file repository here.
-    jcenter()
-    mavenLocal()
+dependencyManagement {
+    imports {
+        mavenBom(SpringBootPlugin.BOM_COORDINATES)
+    }
 }
 
-
 // In this section you declare the dependencies for your production and test code
 dependencies {
     annotationProcessor "org.springframework.boot:spring-boot-configuration-processor"
 
-    compile files('lib/ojdbc8-12.2.0.1.jar')
-
-    compile 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.9.8'
-    compile 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.9.8'
-    compile 'org.springframework.boot:spring-boot-starter-web'
-    compile 'org.springframework.boot:spring-boot-starter-data-jpa'
-    compile 'org.springframework.boot:spring-boot-starter-actuator'
-    compile 'org.springframework.boot:spring-boot-starter-security'
-    compile 'org.springframework.security.oauth:spring-security-oauth2:2.3.6.RELEASE'
-    compile 'org.springframework.security:spring-security-jwt:1.0.10.RELEASE'
-
-    compile 'javax.annotation:javax.annotation-api:1.3.2'
-    compile 'javax.xml.bind:jaxb-api:2.3.0'
-    compile 'com.sun.xml.bind:jaxb-impl:2.3.0'
-    compile 'com.sun.xml.bind:jaxb-core:2.3.0'
-    compile 'javax.activation:activation:1.1.1'
-
-    compile 'org.hibernate:hibernate-java8:5.4.0.Final'
-    compile 'org.springframework.hateoas:spring-hateoas:0.25.0.RELEASE'
-    compile 'io.springfox:springfox-swagger2:2.9.2'
-    compile 'io.springfox:springfox-swagger-ui:2.9.2'
-    compile 'com.zaxxer:HikariCP:3.3.1'
-    compile 'commons-io:commons-io:2.6'
-    compile 'org.apache.commons:commons-lang3:3.8.1'
-    compile 'net.logstash.logback:logstash-logback-encoder:5.2'
-    compile 'com.microsoft.azure:applicationinsights-spring-boot-starter:1.2.0-BETA'
-    compile 'com.microsoft.azure:applicationinsights-logging-logback:2.3.1'
-
-    annotationProcessor 'org.projectlombok:lombok:1.18.4'
+    implementation files('lib/ojdbc8-12.2.0.1.jar')
+
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.9.9'
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.9.9'
```

Comment:
I saw the jdk8 version of the jackson-datatype package and checked it - it is deprecated, as described here 
 https://github.com/FasterXML/jackson-datatype-jdk8 - but might be a bigger job to unpick it?  Happy if we leave this for another PR - I know the point of this one is to make a start on standardising with other services.

Reply:
Well reading https://github.com/FasterXML/jackson-modules-java8#maven-dependencies it seems to me that the github project is deprecated, but the dependencies are still being pushed to the same place, so think this is okay as is.